File: program/print-numbers-from-n-to-1/PrintNumbersFromNTo1.kt
import java.util.*

// Write a Kotlin program to print numbers from n to 1

// Input  : 5
// Output : 5 4 3 2 1

fun main(args: Array<String>) {

    val reader = Scanner(System.`in`)

    print("Enter a number: ")
    val num = reader.nextInt()

    var i = num

    while ( i > 0 )
    {
      print(i.toString() + " ")

      i = i - 1
    } 
}

// Contributed by Technical Amanjeet 

File: program/program/print-numbers-from-n-to-1/PrintNumbersFromNTo1.c
#include <stdio.h>

// Write a C program to print numbers from n to 1

// Input  : 5
// Output : 5 4 3 2 1

void main()
{
    int num;

    printf("Enter number : ");
    scanf("%d", &num);
    
    int i = num;

    while(i != 0)
    {
      printf("%d ", i);

      i = i - 1;
    }
}
